.github/scale-config.yml

```
@@ -86,6 +86,12 @@ runner_types:
     max_available: 200
     disk_size: 150
     is_ephemeral: false
+  linux.g5.12xlarge.nvidia.gpu:
+    instance_type: g5.12xlarge
+    os: linux
+    max_available: 200
```

Why do we need 200 instances?
```suggestion
    max_available: 20
```